src/pages/Route/List.tsx

```diff
@@ -60,22 +62,22 @@ const Page: React.FC = () => {
             onClick={() => history.push(`/routes/${record.id}/edit`)}
             style={{ marginRight: 10 }}
           >
-            编辑
+            {formatMessage({ id: 'route.list.edit' })}
           </Button>
           <Popconfirm
-            title="确定删除该路由吗？"
+            title={formatMessage({ id: 'route.list.delete.confrim' })}
             onConfirm={() => {
               remove(record.id!).then(() => {
-                notification.success({ message: '删除记录成功' });
+                notification.success({ message: formatMessage({ id: 'route.list.delete.success' }) });
                 /* eslint-disable no-unused-expressions */
                 ref.current?.reload();
               });
             }}
-            okText="确定"
-            cancelText="取消"
+            okText={formatMessage({ id: 'route.list.confirm' })}
```

Comment:
Not a good key, we could use global.confirm.ok and global.confirm.cancel instead.